test_env/controller/controller.cfg

```diff
@@ -0,0 +1,29 @@
+[general]
+host = 0.0.0.0
+port = 1502
+actuator_plugins = kvm_io,k8s_replicas,external_api
+metric_source_plugins = monasca
+
+[kvm_io]
+compute_nodes = node1,node2,node3
+key_pair = /home/asperathos/.ssh/asperathos_key
+iops_reference = 50
+bs_reference = 30000000
+default_io_cap = 50
+tunelling = True
+tunnel_ports = 10001,10002,10003
+
+[monasca]
+monasca_endpoint = https://my.monasca.api.endpoint
+username = asperathos-monasca
+password = asperathos-monasca
+project_name = asperathos
+auth_url = https://my.monasca.authentication.url
+api_version = 2_0
+
+[k8s_replicas]
+k8s_manifest = /.kube/config
```

Comment:
This variable now is optional to the use of the 'k8s_replicas' plugin. There is any way to comment this information in this file to make easier to the user to be aware of this info?